Aztec/Classes/Libxml2/DOM/Data/ElementNode.swift

```diff
@@ -234,6 +234,24 @@ public class ElementNode: Node {
     }
 
 
+    /// Returns the ElementNode instance, whenever there's a *single* children, of the specified nodeType (or nil otherwise).
+    ///
+    /// - Parameter type: Type of the 'single children' node to be retrieved.
+    ///
+    /// - Returns: the requested children (if it's the only children in the collection, and if the type matches), or nil otherwise.
+    ///
+    func onlyChild(ofType type: StandardElementType) -> ElementNode? {
+        guard children.count == 1,
+            let onlyChild = children.first as? ElementNode,
+            onlyChild.isNodeType(type)
+        else {
```

Comment:
Our guidelines don't fully apply to this statement, because the `guard` clause has multiple lines / conditions.

Moving the else to the preceding line, would have the sad effect of forcing us to further increase the indentation level of the snippet that goes inside the `else`.

IMO this way we truly improve readability, and avoid increasing indentation. 
I'll submit a formal request to add this to our guideline!

Reply:
I still think this looks very inconsistent with how we format the rest of our code, as we don't separate `else` statement from precedint statements anywhere else.

The fact that auto-indent adds an extra tab should not affect how we format the code.

That said, as this won't affect the logic in the code, we can merge as it is.